### ISSUE TITLE
Nested group within a group

### ISSUE THREAD

**Issue by ilanafle** [philwinne/nstudy_public#72]
If I have a group nested in a group, I don't see it so I can't move items there through the menu.

**Comment by lfdoherty** [comment]
It doesn't appear in the Groups list?

**Comment by ilanafle** [comment]
it appears on the list on the side, but not on the list we use to move stuff from place to place

**Comment by lfdoherty** [comment]
I can't reproduce this - what type of group was it?  Did it have a title?

**Comment by ilanafle** [comment]
yes, both groups have title and I am trying to move a quote from the higher group to the nested group, but the nested group is not an option on the menu.

**Comment by dtruscot** [comment]
I'm also having problems with newly created groups/bookmarks not showing up on the groups list menu
